### PR TITLE
Include table in search index

### PR DESCRIPTION
**Pull request by alikhajeh1**
# Pull request motivation(s)
Fixes https://github.com/algolia/docsearch-configs/issues/4842
Also adds the [Infracost blog](https://www.infracost.io/blog/) in the index as we publish monthly product updates and it'll be useful for users to search them.

### CODE REVIEW

File: configs/infracost.json [hunk 1]
```diff
@@ -19,8 +30,8 @@
     "lvl2": "article h2",
     "lvl3": "article h3",
     "lvl4": "article h4",
-    "lvl5": "article h5, article td:first-child",
-    "text": "article p, article li, article td:last-child"
+    "lvl5": "article h5, article",
+    "text": "article p, article li, article"
```
Comment (shortcuts) [inline configs/infracost.json:34]:
```suggestion
    "lvl5": "article h5",
    "text": "article p, article li, article td"
```

I assume this is what you'd achieve

Comment (alikhajeh1) [reply configs/infracost.json]: I think so, we have `td` that have `code` too, e.g.:
```
<tr><td>Bastion</td><td><code>azurerm_bastion_host</code></td><td></td></tr>
```

Should I include that too? `article p, article li, article td, article td code`

Comment (shortcuts) [reply configs/infracost.json]: It should be enough like that, but feel free to re-open a PR if not

**Review by shortcuts — APPROVED** [review]

File: configs/infracost.json (hunk 1 above)
Comment (alikhajeh1) [reply configs/infracost.json]: Thanks!

**Pull request merged**